changes/ticket32604

```diff
@@ -0,0 +1,7 @@
+  o Minor features (onion services):
+    - Version 3 onion services can now use HiddenServiceExportRendPoint
+      to get first 12 chars of the rendezvous point's fingerprint.
+    - Version 3 onion services can now use HiddenServiceExportInstanceID
+      which enables them to differenciate traffic when multiple tor
+      instances share the same backend.
```

Comment:
Please make similar changes to this change log entry.

Reply:
Fixed in `2582e28`.